manifests/d/Dyad/Dyad/0.21.0/Dyad.Dyad.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with komac v2.13.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: Dyad.Dyad
+PackageVersion: 0.21.0
+InstallerType: exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: install
+ReleaseDate: 2025-09-16
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/dyad-sh/dyad/releases/download/v0.21.0/dyad-0.21.0.Setup.exe
+  InstallerSha256: 119F32A0CF0546E419607DAFA9A9FE710CFA85D5D0CE17F0DCFD16D93D9F3100

```

```suggestion
  InstallerSha256: 119F32A0CF0546E419607DAFA9A9FE710CFA85D5D0CE17F0DCFD16D93D9F3100
  ProductCode: dyad
```

Since there was another requested change of PackageName, figured I would suggest this as well for better matching